app/views/sales/index.html.erb

```diff
@@ -0,0 +1,59 @@
+<h1> <%= @vendor.name %> </h1>
+
+
+<div class="sales_link">
+  <%= link_to new_vendor_sale_path(@vendor.id), class: 'btn btn-success' do %>
+    New Sale
+  <% end %>
+</div>
```

Comment:
a shorter way to write it will be : 

<%= link_to New Sale, new_vendor_sale_path(@vendor.id), class: 'btn btn-success' %>

(no need for " do - end " stuff)